lib/web_apis/api_importer.es6.js

```diff
@@ -127,8 +145,13 @@ foam.CLASS({
                 interfaceName,
                 apiName,
               });
-              this.interfaceDAO.put(webInterface);
-              webInterface.browsers.put(browser);
+              if (this.toCloudDatastore) {
+                this.limiter.enqueue(() => this.interfaceDAO.put(webInterface));
```

Comment:
FOAM question: Could the rate limiter be implemented in a way that makes this transparent to the user, possibly as a DAO that wraps another DAO?